### DISCUSSION TITLE
[Documentation] Mention `forget` in help string of `remove`

### DISCUSSION THREAD

**Discussion by ojdo** [twpayne/chezmoi#3362] [Ideas]
Just a low-key documentation-only feature request: mention the existence of `chezmoi forget` somewhere in the description message of `chezmoi rm -h`, or in a separate "See also" paragraph.

Motivation: I today first wanted to remove a file _only from the source state_. Thinking it might be something `chezmoi rm --source-only` or `chezmoi rm --keep-destination`, I had to resort to Google, leading me to #3182, which mentioned `chezmoi forget`.

**Comment by halostatue** [comment]
This is implemented in #3363 and should be visible with the next release of chezmoi.
